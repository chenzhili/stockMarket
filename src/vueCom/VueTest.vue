<template>
  <div>
    <h1>初始 测试 vue {{value}}</h1>
    <input  v-model="value"/><!--  @input="changeInput" -->
  </div>
</template>
<script>
export default {
  name: 'VueTest',
  data: function() {
    return {
      value: ""
    };
  },
  computed: {},
  methods: {
      changeInput(e) {
        console.log(e.target.value);
        this.value = e.target.value;
      }
  },
  watch: {},
  async mounted() {},
  destroyed() {},
  props: {}
};
</script>
<style lang="scss" scoped>
</style>>