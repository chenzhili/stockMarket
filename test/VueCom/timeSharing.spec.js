import VueTest from '../../src/vueCom/VueTest.vue'
import { mount, shallowMount } from '@vue/test-utils'

let Wrapper;
beforeEach(() => {
    Wrapper = shallowMount(VueTest);
})
describe('简单测试 vue 的组件', () => {
    it('快照测试', () => {
        console.log(Wrapper.find("input").text());
        expect(Wrapper).toMatchSnapshot();
    })
    it('基本事件逻辑测试', () => {

    })
})