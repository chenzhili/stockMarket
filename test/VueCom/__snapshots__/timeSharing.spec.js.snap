// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`简单测试 vue 的组件 快照测试 1`] = `
<div>
  <h1>初始 测试 vue </h1> <input>
</div>
`;
